samples/suit/smp_transfer/prj.conf

```diff
@@ -28,3 +28,6 @@ CONFIG_FLASH=y
 
 # Extended SUIT commands over USER group
 CONFIG_MGMT_SUITFU_GRP_SUIT=y
+
+CONFIG_SUIT_MPI_APP_RECOVERY=y # TODO: do this automatically?
+CONFIG_SUIT_MPI_RAD_RECOVERY=y # TODO: do this automatically?
```

Comment:
You can put this into `sysbuild/smp_transfer_recovery.conf` for the time being. You will then be able to enable this configuration using `FILE_SUFFIX=recovery`

Reply:
I am not sure this would work, as we already need `FILE_SUFFIX=bt` to build with BLE, which would have to be used at the same time.

I actually consider generating MPI-s even if no recovery application is present harmless.

Reply:
Added `default y` for these Kconfig options and removed them from here.